package com.byazt.gq;

import com.byazt.nr.m;
import com.byazt.t.DownloadInfo;
import com.byazt.zz.rh;
import com.bykv.vk.component.ttvideo.player.MediaPlayer;
import org.json.JSONException;
import org.json.JSONObject;

/* JADX INFO: loaded from: classes.dex */
@com.byazt.zqa.c(c = {0, 1, 52, MediaPlayer.MEDIA_PLAYER_OPTION_ENABLE_QCOM_LOW_LATENCY})
public class yp implements rh {
    @Override // com.byazt.zz.rh
    public void c(DownloadInfo downloadInfo, int i, int i2) {
        com.byazt.dd.tt ttVarC = com.byazt.nn.a.c().c(downloadInfo);
        JSONObject jSONObject = new JSONObject();
        try {
            jSONObject.putOpt("reserve_wifi_source", Integer.valueOf(i2));
            jSONObject.putOpt("reserve_wifi_status", Integer.valueOf(i));
        } catch (JSONException e) {
            m.c(e);
        }
        com.byazt.zn.c.c().c("pause_reserve_wifi", jSONObject, ttVarC);
    }
}
